src/ansys/geometry/core/geometry/parameterization.py

```diff
@@ -202,6 +231,139 @@ def get_span(self) -> Real:
+
+        Returns
+        -------
+        bool
+            True if negative False otherwise
+        """
+        return Accuracy.compare_with_tolerance(self.get_span(), 0, tolerance, tolerance)
+
+    @staticmethod
+    def unite(first: "Interval", second: "Interval") -> "Interval":
+        """
+        Return the union of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The union of the two intervals
+        """
+        if first.is_empty():
+            return second
+        if second.is_empty():
+            return first
+        return Interval(min(first.start, second.start), max(first.end, second.end))
+
+    @staticmethod
+    def intersect(first: "Interval", second: "Interval", tolerance: Real) -> "Interval":
+        """
+        Return the intersection of two intervals.
+
+        Parameters
+        ----------
+        first : Interval
+            First interval
+        second : Interval
+            Second interval
+
+        Returns
+        -------
+        Interval
+            The intersection of the two intervals
+        """
+        if first.is_empty() or second.is_empty():
+            return None  # supposed to be empty
+        intersection = Interval(max(first.start, second.start), min(first.end, second.send))
+        if not intersection.is_negative(tolerance):
+            return intersection
+        return None  # supposed to be empty
+
+    def contains_value(self, t: Real, accuracy: Real) -> bool:
+        """
+        Check if the current interval contains value t given the accuracy range.
+
+        Parameters
+        ----------
+        t : Real
+            The value of interest
+        accuracy: Real
+            The accepted range of error since we could be working with floats
+
+        Returns
+        -------
+        bool
+            True if the interval contains the value, false otherwise
+        """
+        if self.is_empty():
+            return False
+        negative_abs_accuracy = -abs(accuracy)
+        if self.start > self.end:
+            if self.start - t < negative_abs_accuracy:
+                return False
+            if t - self.end < negative_abs_accuracy:
+                return False
+        else:
+            if t - self.start < negative_abs_accuracy:
+                return False
+            if self.end - t < negative_abs_accuracy:
+                return False
+        return True
+
+    def contains(self, t: Real) -> bool:
+        """
+        Check if interval contains t using the default length_accuracy constant.
+
+        Parameters
+        ----------
+        t : Real
+            The value of interest
+
+        Returns
+        -------
+        bool
+            True if the interval contains the value, false otherwise
```

Comment:
```suggestion
        Check if the interval contains value ``t`` using the default ``length_accuracy`` constant.

        Parameters
        ----------
        t : Real
            Value of interest.

        Returns
        -------
        bool
            ``True`` if the interval contains the value, ``False`` otherwise.
```